src/components/article-page.jsx

```diff
@@ -24,10 +25,12 @@ const ArticlePage = () => {
         <h2 id="article-author">{singleArticle.article.author}</h2>
         <img id= "article-image" src={singleArticle.article.article_img_url}/>
         <p id="article-body">{singleArticle.article.body}</p>
-
        </article>
+        <ul>
+       <CommentContainer comments={comments} setComments={setComments}/>
```

Comment:
At the moment, if there are no comments, we've got nothing to tell the user this. Could you just add something to account for this situation please?